bin/precompile

```diff
@@ -1,70 +1,50 @@
 #!/usr/bin/env node
-var path = require('path');
+var commander = require('commander');
 var precompile = require('../src/precompile').precompile;
 var Environment = require('../src/environment').Environment;
 var lib = require('../src/lib');
 
-var yargs = require('yargs')
+var cmdpath = null;
 
-    .usage('$0 [-f|--force] [-a|--filters <filters>] [-n|--name <name>] [-i|--include <regex>] [-x|--exclude <regex>] [-w|--wrapper <wrapper>] <path>')
-    .wrap(80)
-
-    .describe('help', 'Display this help message')
-        .boolean('help')
-        .alias('h', 'help')
-        .alias('?', 'help')
-
-    .describe('force', 'Force compilation to continue on error')
-        .boolean('force')
-        .alias('f', 'force')
-
-    .describe('filters', 'Give the compiler a comma-delimited list of asynchronous filters, required for correctly generating code')
-        .string('filters')
-        .alias('a', 'filters')
-
-    .describe('name', 'Specify the template name when compiling a single file')
-        .string('name')
-        .alias('n', 'n')
-
-    .describe('include', 'Include a file or folder which match the regex but would otherwise be excluded. You can use this flag multiple times')
-        .string('include' )
-        .default('include', ['\\.html$', '\\.jinja$'])
-        .alias('i', 'include')
-
-    .describe('exclude', 'Exclude a file or folder which match the regex but would otherwise be included. You can use this flag multiple times')
-        .string('exclude' )
-        .default('exclude', [])
-        .alias('x', 'exclude')
-
-    .describe('wrapper', 'Load a external plugin to change the output format of the precompiled templates (for example, "-w custom" will load a module named "nunjucks-custom")')
-        .string('wrapper')
-        .alias('w', 'wrapper')
-
-    .demand(1);
-
-var argv = yargs.argv;
+function concat(value, previous) {
+  return previous.concat(value);
+}
 
-if (argv.help) {
-    yargs.showHelp();
-    process.exit(1);
+commander
```

Comment:
it would be better to move `concat` function after this `commander` invocation, to keep diff less complicated

Reply:
Some `.option()` call needs `concat()` as an argument, maybe it's not so good to move it after calling, I think.

Reply:
In JavaScript functions get hoisted, so it should work

Reply:
I've moved it. Thank you for your advice.